pkg/generators/models/testdata/cases/validation/api.yaml

```diff
@@ -0,0 +1,62 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+components:
+  schemas:
+    Person:
+      type: object
+      required:
+        - name
+        - gender
+      properties:
+        name:
+          type: string
+          minLength: 2
+          maxLength: 32
```

Comment:
Can we add an array with `minLength`?  Perhaps "favoriteColors" :)

Reply:
added!